src/map/skill.c

```diff
@@ -7809,7 +7816,8 @@ static int skill_castend_nodamage_id(struct block_list *src, struct block_list *
 
 		case TK_HIGHJUMP:
 			{
-				int x,y, dir = unit->getdir(src);
+				int x,y;
```

Comment:
Please declare each variable in its own line.

Reply:
done.